playground/frontend/lib/modules/examples/components/example_list/example_item_actions.dart

```diff
@@ -36,6 +36,8 @@ class ExampleItemActions extends StatelessWidget {
     return Row(
       children: [
         if (example.isMultiFile) multifilePopover,
+        if (example.complexity != Complexity.unspecified)
+          ComplexityWidget(complexity: example.complexity),
```

Comment:
May we consider having a `List<Widget>` variable and applying these if statements before returning `Row`?  My thinking is that it might make the code more readable.  However, I trust your judgement.

Reply:
I don't see a problem with `if`s in a list. There is even a lint to make them right:
https://dart-lang.github.io/linter/lints/prefer_if_elements_to_conditional_expressions.html
The problem here is the line break.

@Malarg you can turn it to a one-liner which is much more readable:

```dart
      children: [
        if (example.isMultiFile) multifilePopover,
        if (example.complexity != null) ComplexityWidget(example.complexity),
        descriptionPopover,
      ],
```

by getting rid of `Complexity.unspecified` and making it nullable, and by turning `ComplexityWidget`'s argument into a positioned one.